dotnet/samples/KernelSyntaxExamples/Example06_TemplateLanguage.cs

```diff
@@ -54,7 +54,7 @@ public static async Task RunAsync()
         Console.WriteLine("--- Rendered Prompt");
         var promptTemplateFactory = new BasicPromptTemplateFactory();
         var promptTemplate = promptTemplateFactory.Create(FunctionDefinition, new PromptTemplateConfig());
-        var renderedPrompt = await promptTemplate.RenderAsync(kernel.CreateNewContext());
+        var renderedPrompt = await promptTemplate.RenderAsync(kernel, kernel.CreateNewContext());
```

Comment:
Do we need to pass this? 

Can't we just get SKContext (while still exists) from within the promptTemplateRender method?

Reply:
Never mind, looking forward to see the upcoming changes on those APIs 🚀